pmm/pmm2-ovf.groovy

```diff
@@ -235,6 +236,7 @@ pipeline {
                             sh '''
                                 FILE=$(ls */*/PMM2-Server-EL9*.ova)
                                 NAME=$(basename ${FILE})
+                                SIZE=\$(du -sh */*/PMM2-Server-EL9*.ova)
```

Comment:
This doesn't seem to be used anywhere

Reply:
Yeah, that was for debugging, will remove it.